In [1]:
!pip install opencv-python

Defaulting to user installation because normal site-packages is not writeable
                                              0.0/38.6 MB ? eta -:--:--
                                              0.0/38.6 MB 2.0 MB/s eta 0:00:19
                                              0.1/38.6 MB 1.7 MB/s eta 0:00:24
                                              0.1/38.6 MB 1.0 MB/s eta 0:00:38
                                              0.2/38.6 MB 1.0 MB/s eta 0:00:38
                                              0.3/38.6 MB 1.3 MB/s eta 0:00:31
                                              0.3/38.6 MB 1.2 MB/s eta 0:00:31
                                              0.4/38.6 MB 1.2 MB/s eta 0:00:31
                                              0.5/38.6 MB 1.3 MB/s eta 0:00:30
                                              0.5/38.6 MB 1.3 MB/s eta 0:00:30
                                              0.6/38.6 MB 1.3 MB/s eta 0:00:30
                                              0.6/38.6 MB 1.

# Tuning Cascade Classifiers
ourClassifier.**detectMultiScale**(input image, **Scale Factor , Min Neighbors**)

- **Scale Factor** Specifies how much we reduce the image size each time we scale. E.g. in face detection we typically use 1.3. This means we reduce the image by 30% each time it’s scaled. Smaller values, like 1.05 will take longer to compute, but will increase the rate of detection.

- **Min Neighbors** Specifies the number of neighbors each potential window should have in order to consider it a positive detection. Typically set between 3-6. It acts as sensitivity setting, low values will sometimes detect multiples faces over a single face. High values will ensure less false positives, but you may miss some faces.